src/base/componentwise.rs

```diff
@@ -40,6 +41,44 @@ impl<N: Scalar, R: Dim, C: Dim, S: Storage<N, R, C>> Matrix<N, R, C, S> {
         res
     }
 
+    /// Computes the component-wise minimum
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use nalgebra::Vector3;
+    ///
+    /// let u = Vector3::new(-1.0, 3.0, 2.0);
+    /// let v = Vector3::new(1.0, 2.0, 3.0);
+    /// assert_eq!(u.component_min(&v), Vector3::new(-1.0, 2.0, 3.0))
+    /// ```
+    pub fn component_min(&self, rhs: &Matrix<N, R, C, S>) -> MatrixMN<N, R, C>
+    where
+        N: Ord,
```

Comment:
I think `Ord` is a too strong of a bound. `PartialOrd` might be a better choice due to `f32` and `f64` not being `Ord`. For minimum, `if a > b { b } else {a}` would be an ok default (prefer first operand if not comparable).

Also, bump :smile: 

I found `inf` and `sup` though the referenced issue, but always expect "component_min" and "component_max" instead.

Reply:
Thanks. `PartialOrd` does work better here.